client/src/Components/ClickableBox.tsx

```diff
@@ -0,0 +1,41 @@
+import {Button, Image, TextBase, TouchableOpacity, View} from "react-native";
+import {getImageByCategory} from "../Utils/getImageByCategory";
+import {replaceEngToHeb} from "../Utils/replaceEngToHeb";
+
+interface ClickableBoxProps {
+    categoryName: string,
+    subCategories: string[]
+}
+
+export const ClickableBox = ({categoryName, subCategories}: ClickableBoxProps) => {
+    const image = getImageByCategory(categoryName);
+    const title = replaceEngToHeb(categoryName);
+    return (
+        <View style={{flexBasis: '50%', alignItems:'center', justifyContent:'center'}}>
+            <TouchableOpacity>
```

Comment:
In general, try to make the `TouchableOpacity` look like the mockup we originally did:
border for the picture, text under it.. whatever you can
In the end of the project there's a section in which we need to make a comparison between the mockup and the end-product, so it's important we stay consistent as much as possible :)